shared/constants/network.js

```diff
@@ -81,3 +88,13 @@ export const CHAIN_ID_TO_NETWORK_ID_MAP = Object.values(
   chainIdToNetworkIdMap[chainId] = networkId;
   return chainIdToNetworkIdMap;
 }, {});
+
+export const NATIVE_CURRENCY_TOKEN_IMAGE_MAP = {
```

Comment:
So this needs to be a map from currency symbols ("ETH', 'TESTETH', 'BNB') to urls, as per gudhatt's comment

> I brought up in a different comment that this image corresponds with the native currency, not the chain ID. Different chains with the same currency will want the same image here (e.g. Mainnet and the Ethereum testnets)

Reply:
oh, but I guess this implmentation has the same effect...

Reply:
and this ensures that if someone adds the BSC network but doesn't specify a symbol, the image will still be correctly selected

Reply:
I don't think this is equivalent. Now the native currency and the image displayed for this currency can get out-of-sync.

For example:
>this ensures that if someone adds the BSC network but doesn't specify a symbol, the image will still be correctly selected

In this situation, the balance would be shown in ETH (the default, if no symbol is entered), but the picture would be of the Binance logo.

Reply:
> In this situation, the balance would be shown in ETH (the default, if no symbol is entered), but the picture would be of the Binance logo.

ah, that's a good point

Reply:
Looking at `ui/app/selectors/send.js`'s `getNativeCurrency` function, `state.metamask.nativeCurrency` is `ETH` on test networks, yet `TEST_ETH_SWAPS_TOKEN_OBJECT.symbol` is `TESTETH`, so that seems weird.  I can update the map to be keyed by symbol easily enough, but the `TESTETH`/`ETH` mismatch will be problematic.

Reply:
so what I think is needed is:
```
export const NATIVE_CURRENCY_TOKEN_IMAGE_MAP = {
  [ETH_SWAPS_TOKEN_OBJECT.symbol]: ETH_SWAPS_TOKEN_OBJECT.iconUrl,
  [TEST_ETH_SWAPS_TOKEN_OBJECT.symbol]: TEST_ETH_SWAPS_TOKEN_OBJECT.iconUrl,
  [BNB_SWAPS_TOKEN_OBJECT.symbol]: TEST_ETH_SWAPS_TOKEN_OBJECT.iconUrl,
};
```
Does that make it less problematic @darkwing?

Reply:
later we can clean it up so that we are not importing these values from swaps, I think the logic here is correct